project2-beat-that/index.html

```diff
@@ -60,9 +68,16 @@
       </ul>
     </nav>
     <!-- The <h1> element defines a large heading. There are 6 pre-set header elements, <h1> to <h6> -->
-    <h1 id="header">🚀 Basics Project 2: Beat That! 🚀</h1>
+    <h1 id="header">🎲 Beat That! 🎲</h1>
     <!-- The <div> tag defines a division or a section in an HTML document, and is commonly use as a "container" for nested HTML elements. -->
     <div class="container">
+      <div style="text-align:center;">
+        Hello! Welcome to Beat That! Click submit to start the game.
+      </p>
+      <p>
+        Create a two-digit number by selecting the order of your dice rolls.
+      </p>
+      <p>The player with the highest number wins! Good luck! 🍀</p></div> 
```

Comment:
Nice manipulation of HTML here! You can also consider giving this div an ID and changing style in CSS instead for uniformity.